### ISSUE TITLE
0.2.3 to 0.2.6 update failed 

### ISSUE THREAD

**Issue by paulochf** [reiinakano/scikit-plot#33]
I've just tried to upgrade the package, but it gave the following error:

```
Collecting scikit-plot
  Using cached scikit-plot-0.2.6.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-7wut1485/scikit-plot/setup.py", line 9, in <module>
        import scikitplot
      File "/tmp/pip-build-7wut1485/scikit-plot/scikitplot/__init__.py", line 5, in <module>
        from scikitplot.classifiers import classifier_factory
      File "/tmp/pip-build-7wut1485/scikit-plot/scikitplot/classifiers.py", line 5, in <module>
        import matplotlib.pyplot as plt
      File "/home/paulo/Programs/repos/pyenv/versions/3.6.1/envs/work-3.6.1/lib/python3.6/site-packages/matplotlib/pyplot.py", line 115, in <module>
        _backend_mod, new_figure_manager, draw_if_interactive, _show = pylab_setup()
      File "/home/paulo/Programs/repos/pyenv/versions/3.6.1/envs/work-3.6.1/lib/python3.6/site-packages/matplotlib/backends/__init__.py", line 32, in pylab_setup
        globals(),locals(),[backend_name],0)
      File "/home/paulo/Programs/repos/pyenv/versions/3.6.1/envs/work-3.6.1/lib/python3.6/site-packages/matplotlib/backends/backend_tkagg.py", line 6, in <module>
        from six.moves import tkinter as Tk
      File "/home/paulo/Programs/repos/pyenv/versions/3.6.1/envs/work-3.6.1/lib/python3.6/site-packages/six.py", line 92, in __get__
        result = self._resolve()
      File "/home/paulo/Programs/repos/pyenv/versions/3.6.1/envs/work-3.6.1/lib/python3.6/site-packages/six.py", line 115, in _resolve
        return _import_module(self.mod)
      File "/home/paulo/Programs/repos/pyenv/versions/3.6.1/envs/work-3.6.1/lib/python3.6/site-packages/six.py", line 82, in _import_module
        __import__(name)
      File "/home/paulo/Programs/repos/pyenv/versions/3.6.1/lib/python3.6/tkinter/__init__.py", line 36, in <module>
        import _tkinter # If this fails your Python may not be configured for Tk
    ModuleNotFoundError: No module named '_tkinter'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-7wut1485/scikit-plot/
```

Unfortunately, I don't know how to debug this problem. If you need some info, please don't hesitate to ask!

**Comment by reiinakano** [comment]
Hi! I'm not sure why this would happen if you're just upgrading the package, but have you tried checking if you have tkinter installed? tkinter is a dependency for matplotlib.

Hmm this link https://stackoverflow.com/questions/36327134/matplotlib-error-no-module-named-tkinter seems to have a solution for Python 3.

**Comment by paulochf** [comment]
I had matplotlib installed when I updated, and right now I can make plots with it. Does this mean that tkinter is installed?

**Comment by reiinakano** [comment]
Yup, it probably means you have tkinter. I'm not aware of any change from 0.2.3 to 0.2.6 that would cause this to be honest. 

Perhaps uninstall and reinstall scikit-plot?

**Comment by paulochf** [comment]
Tried already, no avail

**Comment by reiinakano** [comment]
That's really odd. What's your Os?

**Comment by paulochf** [comment]
```
$ uname -a
Linux cookiemonster 4.4.0-79-generic #100-Ubuntu SMP Wed May 17 19:58:14 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by paulochf** [comment]
Just tried again to see if I could have the cookie.

```bash
$ sudo apt install python3-tk
[sudo] password for paulo: 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
python3-tk is already the newest version (3.5.1-1).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.

$ pip install scikit-plot
Collecting scikit-plot
  Using cached scikit-plot-0.2.6.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-es9lj1z1/scikit-plot/setup.py", line 9, in <module>
        import scikitplot
      File "/tmp/pip-build-es9lj1z1/scikit-plot/scikitplot/__init__.py", line 5, in <module>
        from scikitplot.classifiers import classifier_factory
      File "/tmp/pip-build-es9lj1z1/scikit-plot/scikitplot/classifiers.py", line 5, in <module>
        import matplotlib.pyplot as plt
      File "/home/paulo/Programs/repos/pyenv/versions/3.6.1/envs/scorer-lab/lib/python3.6/site-packages/matplotlib/pyplot.py", line 115, in <module>
        _backend_mod, new_figure_manager, draw_if_interactive, _show = pylab_setup()
      File "/home/paulo/Programs/repos/pyenv/versions/3.6.1/envs/scorer-lab/lib/python3.6/site-packages/matplotlib/backends/__init__.py", line 32, in pylab_setup
        globals(),locals(),[backend_name],0)
      File "/home/paulo/Programs/repos/pyenv/versions/3.6.1/envs/scorer-lab/lib/python3.6/site-packages/matplotlib/backends/backend_tkagg.py", line 6, in <module>
        from six.moves import tkinter as Tk
      File "/home/paulo/Programs/repos/pyenv/versions/3.6.1/envs/scorer-lab/lib/python3.6/site-packages/six.py", line 92, in __get__
        result = self._resolve()
      File "/home/paulo/Programs/repos/pyenv/versions/3.6.1/envs/scorer-lab/lib/python3.6/site-packages/six.py", line 115, in _resolve
        return _import_module(self.mod)
      File "/home/paulo/Programs/repos/pyenv/versions/3.6.1/envs/scorer-lab/lib/python3.6/site-packages/six.py", line 82, in _import_module
        __import__(name)
      File "/home/paulo/Programs/repos/pyenv/versions/3.6.1/lib/python3.6/tkinter/__init__.py", line 36, in <module>
        import _tkinter # If this fails your Python may not be configured for Tk
    ModuleNotFoundError: No module named '_tkinter'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-es9lj1z1/scikit-plot/
```

**Comment by reiinakano** [comment]
Hi @paulochf , is there still an issue here?

**Comment by paulochf** [comment]
Hey, @reiinakano! I need to try to install it again. I'll try during the next days and will give you an update, ok?

**Comment by paulochf** [comment]
Installation went fine, fixed for 0.2.7.
